forc-plugins/forc-node/src/ignition/cmd.rs

```diff
@@ -0,0 +1,30 @@
+use crate::{
+    consts::{DEFAULT_PEERING_PORT, DEFAULT_PORT, MAINNET_BOOTSTRAP_NODE},
+    util::DbConfig,
+};
+use clap::Parser;
+use std::{net::IpAddr, path::PathBuf};
+
+#[derive(Parser, Debug, Clone)]
+pub struct IgnitionCmd {
```

Comment:
nit: it looks like the options are the same for `TestnetCmd`, it's just the defaults that differ. We could have a common struct for the options that are the same and use `flatten`